### ISSUE TITLE
Keyboard navigation on results page

### ISSUE THREAD

**Issue by BITS-Editor** [bonartm/quizdown-js#27]
It seems that is not possible navigate to and activate the links to the individual questions on the results page by keyboard. This is crucial for  accessibility requirements.

It would be great if the links to the questions on the results page could be accessed by Shift-Tab etc.
![BITS-Screenie](https://user-images.githubusercontent.com/7092938/174799813-c168d17a-21fc-4c37-a760-743b87378a58.GIF)

**Comment by BITS-Editor** [comment]
Any updates?

**Comment by bonartm** [comment]
No, I don't have time to work on that. I'll leave the issue open for someone else to help us with it.
